### PR TITLE
[DEVELOPER-3286] Synced current menu layout into Drupal layout

### PR DESCRIPTION
**Pull request by robpblake**
This PR syncs the current Awestruct menu layout into the Drupal layout.

### CODE REVIEW

**Comment by robpblake** [comment]
@RubyGemTSL This should help us on the way to getting the tests to pass once Jenkins is back up and running as I've synced the current Awestruct site menu with the Drupal one.

cheers,
Rob

**Comment by LightGuard** [comment]
Were there other changes besides just the menu that need to be changed as well?

**Comment by robpblake** [comment]
@LightGuard Not that I am currently aware of, but once we get further into the testing it may throw further things up.

**Comment by LightGuard** [comment]
Closing until Jenkins comes back

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig [hunk 1]
```diff
@@ -97,74 +105,78 @@
...
+                              <a class="hide-on-mobile" href="/products/fuse/overview">Red Hat JBoss Fuse</a>
+                              <a class="hide-on-mobile" href="/products/bpmsuite/overview">Red Hat JBoss BPM Suite</a>
+                              <a class="hide-on-mobile" href="/products/brms/overview">Red Hat JBoss BRMS</a>
                             </div>
+                          <div class="sub-nav-group">
+                            <a class="heading" href="/products#mobile">Mobile</a>
+                            <a class="hide-on-mobile" href="/products/mobileplatform/overview">Red Hat Mobile Application Platform</a>
+                          </div>
+                          <div class="sub-nav-group">
+                            <a class="heading" href="/products#cloud_products">Cloud</a>
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig]: This isn't the same as production right now. In production it's called "Private Cloud" and the link should go to "/products#private_cloud.

**Comment by LightGuard** [comment]
All looks good save the one link issue I found. Note in production the "Private Cloud" link doesn't go where it should in the page.

**Comment by robpblake** [comment]
@LightGuard Fixed the incorrect link. Thanks for spotting it.

**Pull request merged**